src/build/template.html

```diff
@@ -2,7 +2,7 @@
 <html lang="{process.env.LOCALE}" dir="{process.env.LOCALE_DIRECTION}">
 <head>
   <meta charset='utf-8' >
-  <meta name="viewport" content="width=device-width, viewport-fit=cover">
+  <meta name="viewport" content="width=device-width">
```

Comment:
`viewport-fit=cover` was added in this PR (https://github.com/nolanlawson/pinafore/pull/1199) to fix design issues with iPhones that have a notch for the camera. Why are we removing it?

Reply:
I was experimenting with status bar values, removing that code fixed status bar disappearing.

I only tested on home button iPhone, so it may require testing on notched iphone.

It's been 3 years though. And i think that bug's been fixed since another projects don't have it/and works well :)

Reply:
I tried to go back through old issues to figure out what this does (https://github.com/nolanlawson/pinafore/pull/1199) (https://github.com/nolanlawson/pinafore/pull/1197#issuecomment-490758966), and it looks like [Apple doesn't recommend it anymore](https://stackoverflow.com/a/60939741). It looks like this was designed to fix the Snackbar (the thing that slides up from the bottom when there's an update). I don't have a notched iPhone that I can test, but I guess I can grab an iOS Simulator to try to see what this is doing.

Reply:
Just tested, and I don't see any difference with or without the `viewport-fit=cover`, on notched or non-notched phones. So yeah I think we can remove this.

<details><summary>screenshots of various iphones</summary>


![Screenshot from 2022-04-10 09-25-08](https://user-images.githubusercontent.com/283842/162629713-9073e7db-a55d-44c2-bbd0-ac0d86c09c76.png)
![Screenshot from 2022-04-10 09-26-34](https://user-images.githubusercontent.com/283842/162629715-00bdeb1a-757b-4df7-bc49-e361412a668f.png)
![Screenshot from 2022-04-10 09-27-54](https://user-images.githubusercontent.com/283842/162629717-4ce617e7-103d-4515-a787-20ed702c0cbe.png)
![Screenshot from 2022-04-10 09-28-42](https://user-images.githubusercontent.com/283842/162629719-f3fb540f-3c31-44d2-bc80-e872808fbb23.png)
![Screenshot from 2022-04-10 09-30-28](https://user-images.githubusercontent.com/283842/162629721-1cefa2c5-bfc3-48d5-a427-0042e5508760.png)
![Screenshot from 2022-04-10 09-22-17](https://user-images.githubusercontent.com/283842/162629710-3d92e877-e5f2-4486-ac54-70b4b1915a1f.png)

</details>